### DISCUSSION TITLE
VOR modules

### DISCUSSION THREAD

**Discussion by ghost** [eh2k/squares-and-circles#79] [Q&A]
Trying to get s&C working on a plumaudio 1U onc. The teensy boots up and all the controls work fine, but wont calibrate. Ive hunted through the repo for where the initial DAC calibration is called from, so I can add a greater offset. Any ideas (sorry I know its not really an issue)

**Comment by eh2k** [comment]
Hello @kurtjoy,
since the teensy4 does not have a DAC on pin A14, VOR cannot actually work. Possible you can set the normal behavior using jumpers. Maybe the description of the DAC-Voltage-Rage-Mod is helpful (https://github.com/eh2k/squares-and-circles#optional-dac-voltage-range-mod--5v5v-range)

The calibration and hardware specific implementation is currently only provided as a closed-source library - without the possibility for adjustments (https://github.com/eh2k/squares-and-circles#license)
